dbld/images/centos-7.dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM centos:7
 LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, Balazs Scheidler <balazs.scheidler@oneidentity.com>"
+ENV OS_DISTRIBUTION=centos
+ENV RELEASE_CODE_NAME=7
```

Comment:
Seems nitpicking, but without context the name `RELEASE_CODE_NAME` doesn't really suggesting what should be the content.
And it's really tied to `OS_DISTRIBUTION` (e.g. it's release 7 of centos distro), so we should name it to emphasize this.
Also, we already have syslog-ng release related variables (see `RELEASE_DIR`), this would really mix things up.

What do you think about changing it to `OS_RELEASE_CODE_NAME` or `DISTRIBUTION_CODE_NAME` or OS_DISTRIBUTION_CODE_NAME`?

Reply:
At the beginning of the development I thought how to name it, and the reason of why I named to `RELEASE_CODE_NAME` is I checked the official debian site and they are referring to release code.
But your suggestion is better :) so I changed to `OS_DISTRIBUTION_CODE_NAME`